plugin-server/src/worker/vm/upgrades/historical-export/export-historical-events.ts

```diff
@@ -31,15 +31,15 @@ const OLD_TIMESTAMP_CURSOR_KEY = 'old_timestamp_cursor'
 
 const INTERFACE_JOB_NAME = 'Export historical events'
 
-export async function addHistoricalEventsExportCapability(
+export function addHistoricalEventsExportCapability(
     hub: Hub,
     pluginConfig: PluginConfig,
     response: PluginConfigVMInternalResponse<PluginMeta<ExportHistoricalEventsUpgrade>>
-): Promise<void> {
+): void {
     const { methods, tasks, meta } = response
 
     // we can void this as the job appearing on the interface is not time-sensitive
-    await addPublicJobIfNotExists(hub.db, pluginConfig.plugin_id, INTERFACE_JOB_NAME, {})
+    void addPublicJobIfNotExists(hub.db, pluginConfig.plugin_id, INTERFACE_JOB_NAME, {})
```

Comment:
https://www.youtube.com/watch?v=7nqcL0mjMjw

Reply:
hahah well this was a voided promise before which I turned to `await` during some experimentation.

I've done some work on this here though: #8596